hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-common/src/test/java/org/apache/hadoop/yarn/server/federation/utils/FederationPoliciesTestUtil.java

```diff
@@ -244,7 +245,7 @@ public static FederationStateStoreFacade initFacade(
       List<SubClusterInfo> subClusterInfos, SubClusterPolicyConfiguration
       policyConfiguration) throws YarnException {
     FederationStateStoreFacade goodFacade = FederationStateStoreFacade
-        .getInstance();
+        .getInstance(new Configuration());
```

Comment:
Can we have an empty getInstance() that does the new Configuration?

Reply:
Thanks for your suggestion! I will provide an empty getInstance.